src/App.jsx

```diff
@@ -55,6 +56,13 @@ const MainAppRoutes = () => (
         <AccountSettings />
       </BaseLayout>
     </SentryRoute>
+    {!config.IS_SELF_HOSTED && (
+      <SentryRoute path="/sync" exact>
```

Comment:
Why didn't you move the sync page to base layout?

Reply:
Not sure I understand what you mean? The sync page is wrapped by the base layout.

Reply:
Ohh I see what you mean here now, see [here](https://github.com/codecov/gazebo/pull/2235#issuecomment-1718021025)

Reply:
@nicholas-codecov FYI - we'll want this available in enterprise environments as well (which I think is what `IS_SELF_HOSTED` means?)

Reply:
I was told/under the impression that this will be released later, not right away.

Reply:
It's also currently behind a feature flag so it won't work anyways on self-hosted, even if this was removed I think*

Reply:
Looking to release Okta login for a new dedicated cloud deployment which would also need this `/sync` page available.  Is it possible to remove the feature flag as well?  There's nothing other than the API that links to this new page, right?

Reply:
Alternatively could the feature flag be passed in as a build time config env or gate it to app_env enterprise?

Reply:
Once cloud has been tested and working a day or two, I will remove the flag, and then at the same time I'll remove the self hosted checks. 

Also to release Okta work will have to be done to the enterprise login page to even support it, so that has to be done before any release to dedicated cloud is done.

Reply:
Something else I forgot to mention, is that we also have to fix the URLs that Gazebo currently uses to login/logout users, since they are currently using the base url not api url.